rbac/src/test/java/org/jboss/hal/testsuite/test/rbac/ServerGroupDetailTestCase.java

```diff
@@ -38,15 +38,25 @@
     private ServerGroupDetailPage serverGroupDetailPage;
 
     @Test
-    public void mainAdministratorOtherMonitor() {
+    public void modificationButtonsNotVisibleForMonitor() {
+        Authentication.with(browser).authenticate(RbacRole.MAIN_ADMINISTRATOR_OTHER_MONITOR);
+
+        startOnRuntimePage();
+        goToServerGroupDetailPage("other-server-group");
+
+        checkVisibilityOfButtons(false);
+    }
+
+    @Test
+    public void modificationButtonsVisibleForAdministrator() {
         Authentication.with(browser).authenticate(RbacRole.MAIN_ADMINISTRATOR_OTHER_MONITOR);
 
         startOnRuntimePage();
         goToServerGroupDetailPage("other-server-group");
```

Comment:
Being at `other-server-group` I suggest to test `checkVisibilityOfButtons(false)` so you would test the scenario that switching between server groups really means changing visibility of buttons. Than the `modificationButtonsNotVisibleForMonitor` can be removed and `modificationButtonsVisibleForAdministrator` renamed. 
Sorry for my previous vague comment about negative test. I hope now it's clear.